hr_employee_age/models/hr_employee.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2015 Salton Massally (<smassally@idtlabs.sl>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    age = fields.Integer(string="Age", readonly=True, compute="_compute_age")
+
+    @api.multi
+    @api.depends("birthday")
+    def _compute_age(self):
+        for record in self:
+            age = 0
+            if record.birthday:
+                age = relativedelta(
+                    fields.Date.to_date(fields.Date.today()),
+                    fields.Date.to_date(record.birthday),
```

Comment:
`fields.Date.today()` and `record.birthday` are already dates, why are you casting them?

Reply:
I just blindly copied the instructions regarding dates on the OCA migration page which state that `fields.Date/Datetime.from_string` (which where present) should be replaced by `fields.Date/Datetime.to_date`. But indeed, both are dates so this can be removed!